latest/index.bs

```diff
@@ -124,8 +124,8 @@ multiple levels of resolutions and optionally associated labels.
     ├── n                     # The name of the array is arbitrary with the ordering defined by
     │   │                     # by the "multiscales" metadata, but is often a sequence starting at 0.
     │   │
-    │   ├── .zarray           # All image arrays are 5-dimensional
-    │   │                     # with dimension order (t, c, z, y, x).
+    │   ├── .zarray           # All image arrays are up to 5-dimensional with dimension order (t, c, z, y, x).
+    │   │                     # An image without time dimension has axes (c, z, y, x) etc.
```

Comment:
Is a 3D array interpreted as `(z, y, x)`?

Reply:
If we promote "axes" to MUST, then the interpretation always depends on the metadata, i.e. on the values in "axes".